src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/SimpleRecordsFetcherFactory.java

```diff
@@ -44,7 +45,7 @@ public GetRecordsCache createRecordsFetcher(GetRecordsRetrievalStrategy getRecor
                             .setDaemon(true)
                             .setNameFormat("prefetch-cache-" + shardId + "-%04d")
                             .build()),
-                    idleMillisBetweenCalls);
+                    idleMillisBetweenCalls, metricsFactory, "processTask");
```

Comment:
It should be ProcessTask and not processTask.
Add the 2 new parameters in new lines each.